### PR TITLE
docs: contribute connector example

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

contribute connector guides with examples.

## Testing

N/A

### CODE REVIEW

File: docs/docs/recipes/create-your-connector/connector-implementation-guide.mdx [hunk 1]
```diff
@@ -3,44 +3,261 @@ sidebar_label: Guide to implement connectors
...
+
+### `getUserInfo`
+
+`getUserInfo` fetches user information with access token got in previous step.
+
+`userInfoEndpoint` is the endpoint which is used to get user information.
+
+You may check official documents to find specific user information that can be accessed at user info endpoint and corresponding scope.
+
+`id` assigned by the connector's service provider (in this case, GitHub) is required; other information is optional. If you can get `email` or `phone` from the user profile, MAKE SURE that they are "verified" since they will be used to receive passcodes in passwordless flows. You also need to align the returning keys with the fields name in [Logto user profile](../../references/users/README.md#basic-data).
```
Comment (wangsijie) [inline docs/docs/recipes/create-your-connector/connector-implementation-guide.mdx:98]: The reason why they should be verified is not accurate, it's hard to describe, may be delete the "since"?

File: docs/docs/recipes/create-your-connector/connector-implementation-guide.mdx [hunk 2]
```diff
@@ -3,44 +3,261 @@ sidebar_label: Guide to implement connectors
...
+- `endpoint` is the endpoint your API calls connect to.
+- `config` contains `templates` (for sending passcode in different user flows), `clientId` and `clientSecret` (to access to API requests).
+
+```typescript
+const sendMessage = async (data, inputConfig) => {
+  const { to, type, payload } = data;
+  const config = inputConfig ?? (await getConfig(defaultMetadata.id));
+  validateConfig<AliyunDmConfig>(config, aliyunDmConfigGuard);
+  const { accessKeyId, accessKeySecret, accountName, fromAlias, templates } = config;
+  const template = templates.find((template) => template.usageType === type);
```
Comment (wangsijie) [inline docs/docs/recipes/create-your-connector/connector-implementation-guide.mdx:161]: Do we need to add some explanation on "template"?

Comment (wangsijie) [reply docs/docs/recipes/create-your-connector/connector-implementation-guide.mdx:161]: Or add a link to the full code

**Review by charIeszhao — APPROVED** [review]

File: docs/docs/recipes/create-your-connector/connector-implementation-guide.mdx (hunk 1 above)
Comment (charIeszhao) [reply docs/docs/recipes/create-your-connector/connector-implementation-guide.mdx]: Maybe they don't necessarily have to be "verified" any more, since we can use the dedicated "email_verified" and "phone_verified" properties?

**Review by wangsijie — APPROVED** [review]

File: docs/docs/recipes/create-your-connector/connector-implementation-guide.mdx (hunk 1 above)
Comment (darcyYe) [reply docs/docs/recipes/create-your-connector/connector-implementation-guide.mdx]: we added "email/phone_verified" to the return value of SDK to align with the protocol, and these two values will always be `true` per our current system/DB design. Will revisit this after "profile fulfilling" ends to see if unverified email/phone should be supported.
TL;DR: keep returning verified email/phone from users' profiles. cc: @charIeszhao
